.gitignore

```diff
@@ -8,3 +8,5 @@ h2spec
 # Files generated by honggfuzz
 hfuzz_target
 hfuzz_workspace
+
+**/.idea
```

Comment:
@seanmonstar do you care if there are IDE-specific ignore rules?

Reply:
Hm, so far we haven't, since this can be one on an individual's machine, by putting them in `~/.gitignore`.

Reply:
I can move this out. Will address it along with other comments once all stackholders' opinions are received